lib/java/test/org/apache/thrift/test/DeserializationBenchmark.java

```diff
@@ -0,0 +1,165 @@
+package org.apache.thrift.test;
```

Comment:
ASF license header missing

Reply:
sorry for the oversight. added license header comment.